int16 sensor1
int16 sensor2
int16 sensor3
int16 sensor4